client/Components/OAuth.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+
+class OAuth extends Component {
+  state = {}
+  render() {
+    const loggedIn = false
+    return (
+      <React.Fragment>
```

Comment:
i think we should agree if we'd use `<></>` or `<React.Fragment></React.Fragment>`. right now, this is already inconsistent with what's in App.jsx